src/Octokit.Webhooks/Converter/ChangesFieldValueChangeConverter.cs

```diff
@@ -0,0 +1,57 @@
+namespace Octokit.Webhooks.Models.ProjectsV2ItemEvent;
+
+public class ChangesFieldValueChangeConverter : JsonConverter<ChangesFieldValueChangeBase>
+{
+    public override ChangesFieldValueChangeBase? Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        switch (reader.TokenType)
+        {
+            case JsonTokenType.StartObject:
+                var changeObject = JsonSerializer.Deserialize<ChangesFieldValueChange>(ref reader, options);
+                return changeObject;
+            case JsonTokenType.String:
+            case JsonTokenType.True:
+            case JsonTokenType.False:
+                return new ChangesFieldValueScalarChange { StringValue = reader.GetString() };
+            case JsonTokenType.Null:
+                return null;
+            case JsonTokenType.Number:
+                return new ChangesFieldValueScalarChange { NumericValue = reader.GetDecimal() };
+            case JsonTokenType.None:
+            case JsonTokenType.EndObject:
+            case JsonTokenType.StartArray:
+            case JsonTokenType.EndArray:
+            case JsonTokenType.PropertyName:
+            case JsonTokenType.Comment:
```

Comment:
`default` catches all these, no need to explicitly specify them.

Reply:
Would you prefer it that way? In that case, with pattern matching, no analyzer can spell out what all the cases would be, so it can't give a diagnostic. The flip side of this way is that if a new JsonTokenType was added that was in some way relevant, you'd have no warning when using pattern matching cases, but would know immediately using the code as it stands right now.

Reply:
I can't see a new JsonTokenType being added, and if GitHub allows ChangesField to accept new types then this code will need to be updated regardless of how its written.